Composer/packages/server-workers/src/templateInstallation.worker.ts

```diff
@@ -8,23 +8,22 @@ import yeoman from 'yeoman-environment';
 import TerminalAdapter from 'yeoman-environment/lib/adapter';
```

Comment:
remove commented code
a bunch of awaits down below are not necessary, linter shows them.

Reply:
@hatpick my recent change upgrade the yeoman-environment package, but the @types package has not been updated upstream- so the linter may warn about unnecessary `await`s but the methods being invoked do return promises.